packages/project-access/src/project/info.ts

```diff
@@ -203,3 +203,24 @@ export async function getProjectType(projectRoot: string): Promise<ProjectType>
     }
     return capType;
 }
+
+/**
+ * Reads the package.json file and returns its content.
+ *
+ * @param {Editor} fs - The file system editor.
+ * @param {string} packageJsonPath - The path to the package.json file.
+ * @returns {Package} The content of the package.json file.
+ */
+export function getPackageJson(fs: Editor, packageJsonPath: string): Package {
```

Comment:
the usual pattern is to have `fs: Editor` as optional paramter, if provided it will be read from there, otherwise from the file system.